extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "Enables users to develop and publish database schemas for MSSQL Databases",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
version 1.3.0 hasn't been released yet, so the version doesn't need to be bumped